rulesets/data/es_zerh_lookup.csv

```diff
@@ -0,0 +1,42 @@
+type,program,window_class,window_type,1A,1B,1C,2A,2B,2C,3A,3B,3C,4A,4B,4C,5A,5B,5C,6A,6B,6C,7,8
```

Comment:
I suggest replacing window_class and window_type with a generic subtype (that can be used across component types).